view/frontend/templates/breadcheckout/view.phtml

```diff
@@ -28,7 +28,7 @@
         document.configureButton = function (items) {
             var buttonOnProduct     = <?= /* @noEscape */ $block->getIsButtonOnProduct(); ?>;
             var buttonCss           = '<?= /* @noEscape */ $block->getButtonDesign(); ?>';
-            var cartSizeFinancing   = <?= /* @noEscape */ $block->getCartSizeFinancingJson(); ?>;
+            var isFinancing   = <?= /* @noEscape */ $block->getFinancingJson(); ?>;
```

Comment:
I think isFinancing should be changed to financingInfo or something, since it's not just a boolean but an object of data around financing

Reply:
Yes, that makes more sense